src/main/java/core/basesyntax/CreateFigure.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class CreateFigure {
```

Comment:
```suggestion
public class FigureProducer {
```
classes are usually nouns not verbs